torch/_prims/context.py

```diff
@@ -248,10 +248,21 @@ def _is_func_unsupported_nvfuser(
 
 class TorchRefsNvfuserCapabilityMode(TorchRefsMode):
     def __init__(self, *, skip_ops=()):
+        aten_ops_to_skip = (
+            "aten.transpose.int",
+            "aten.t.default",
+            "aten.view.default",
+            "aten.unsqueeze.default",
+            "aten.permute.default",
+            "aten._log_softmax.default",
+            "aten._log_softmax_backward_data.default",
+            "aten.expand.default",
```

Comment:
Can we add `aten.col2im` in the list because of #86332

Reply:
This actually doesn't help here... it gets decomposed during fake tensor tracing and we hit this prior to our decomposition. :cry: